examples/advanced/tidb-cluster.yaml

```diff
@@ -1,295 +1,865 @@
+      # mysqlNodePort: 30020
+      # # whether to export the status port, defaults to true
+      # exposeStatus: true
+      # # which NodePort to expose 10080 (status) port of tidb-server, only effective when type=LoadBalancer/NodePort and exposeStatus=true
+      # statusNodePort: 30040
+
+    ###############################
+    # Advanced TiDB Configuration #
+    ###############################
+
+    ## The following block overwrites TiDB cluster-level configurations in `spec`
+    # version: "v4.0.8"
+    # imagePullPolicy: IfNotPresent
+    # imagePullSecrets: secretName
+    # hostNetwork: false
+    # serviceAccount: advanced-tidb-tidb
+    # priorityClassName: system-cluster-critical
+    # schedulerName: tidb-scheduler
+    # nodeSelector:
+    #   app.kubernetes.io/component: tidb
+    # annotations:
+    #   node.kubernetes.io/instance-type: some-vm-type
+    # tolerations:
+    #   - effect: NoSchedule
+    #     key: dedicated
+    #     operator: Equal
+    #     value: tidb
+    # configUpdateStrategy: RollingUpdate
+    # statefulSetUpdateStrategy: RollingUpdate
+
+    ## List of environment variables to set in the container
+    ## Note that the following env names cannot be used and will be overwritten by TiDB Operator builtin envs
+    ##   - NAMESPACE
+    ##   - TZ
+    ##   - SERVICE_NAME
+    ##   - PEER_SERVICE_NAME
+    ##   - HEADLESS_SERVICE_NAME
+    ##   - SET_NAME
+    ##   - HOSTNAME
+    ##   - CLUSTER_NAME
+    ##   - POD_NAME
+    ##   - BINLOG_ENABLED
+    ##   - SLOW_LOG_FILE
+    ## Ref: https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/
+    # env:
+    # - name: MY_ENV_1
+    #   value: value1
+    # - name: MY_ENV_2
+    #   valueFrom:
+    #     fieldRef:
+    #       fieldPath: status.myEnv2
+
+    ## Custom sidecar containers can be injected into the TiDB pods,
+    ## which can act as a logging/tracing agent or for any other use case
+    # additionalContainers:
+    # - name: myCustomContainer
+    #   image: ubuntu
+
+    ## custom additional volumes in TiDB pods
+    # additionalVolumes:
+    # # this must match `name` in `additionalVolumeMounts`
+    # - name: nfs
+    #   nfs:
+    #     server: 192.168.0.2
+    #     path: /nfs
+
+    ## custom additional volume mounts in TiDB pods
+    # additionalVolumeMounts:
+    # # this must match `name` in `additionalVolumes`
+    # - name: nfs
+    #   mountPath: /nfs
+
+    ## Optional duration in seconds the pod needs to terminate gracefully. May be decreased in delete request.
+    ## Ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#hook-handler-execution
+    # terminationGracePeriodSeconds: 30s
+
+    ## PodSecurityContext holds pod-level security attributes and common container settings.
+    ## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+    # podSecurityContext:
+    #   sysctls:
+    #   - name: net.ipv4.tcp_syncookies
```

Comment:
Please use the ones [here](https://github.com/pingcap/tidb-operator/blob/master/examples/aws/tidb-cluster.yaml#L64), similar comment for tikv.